pep-0520.txt

```diff
@@ -410,19 +410,13 @@ discovery is almost entirely an implementation detail.
 References
 ==========
 
-.. [impl] issue #24254
-   (https://bugs.python.org/issue24254)
-
-.. [nick_concern] Nick's concerns about mutability
-   (https://mail.python.org/pipermail/python-dev/2016-June/144883.html)
-
-.. [orig] original discussion
+* Original discussion
    (https://mail.python.org/pipermail/python-ideas/2013-February/019690.html)
 
-.. [followup1] follow-up 1
+* Follow-up 1
    (https://mail.python.org/pipermail/python-dev/2013-June/127103.html)
```

Comment:
```suggestion
* `Follow-up 1
  <https://mail.python.org/pipermail/python-dev/2013-June/127103.html>`__
```

Reply:
And for "Original discussion" too? If you add a suggestion I'll batch them together :)

Reply:
Had to bifurcate as it covers added and removed lines.

A